components/feedstock/thumbnail.js

```diff
@@ -1,8 +1,39 @@
 import { Badge } from '@carbonplan/components'
 import { alpha } from '@theme-ui/color'
-import { Box } from 'theme-ui'
+import { Box, Flex } from 'theme-ui'
 
-export const Thumbnail = ({ url, color, demo }) => {
+const Tags = ({ tags, sx }) => {
+  // Sort tags alphabetically
+  const sortedTags = tags.sort((a, b) => a.localeCompare(b))
+  return (
+    <Flex
+      sx={{
+        columnGap: 2,
+        rowGap: 1,
+        flexWrap: 'wrap',
+        ...sx,
+      }}
+    >
+      {sortedTags.map((tag) => (
+        <Badge
+          key={tag}
+          sx={{
+            fontSize: [0, 0, 0, 1],
+            height: ['24px', '24px', '24px', '24px'],
+            lineHeight: ['24px', '24px', '24px', '24px'],
```

Comment:
```suggestion
            fontSize: [0, 0, 0, 1],
            height: ['22px', '22px', '22px', '24px'],
            lineHeight: ['22px', '22px', '22px', '24px'],
```
Again, I think you might want to special-case the largest breakpoint here (as you've already done for `fontSize`.